src/schemas/json/bamboo-spec.json

```diff
@@ -446,9 +515,11 @@
           "additionalProperties": false,
           "properties": {
             "manual": {
+              "description": "Will await for execution by user",
               "type": "boolean"
             },
             "final": {
+              "description": "Will await for execution by user",
```

Comment:
Will be executed regardless of other stages status

Reply:
Thanks, fixed

Reply:
👍